hellonode/server.js

```diff
@@ -15,11 +15,19 @@
  */
 
 var http = require('http');
+var os = require('os');
+
+process.on('SIGINT', function() {
+  console.log('shutting down...');
+  process.exit();
+});
 
 var handleRequest = function(request, response) {
   console.log('Received request for URL: ' + request.url);
   response.writeHead(200);
-  response.end('Hello, World!');
+  response.end('Hello, World!\nHostname: ' + os.hostname() + '\n');
```

Comment:
you could use a template string here
```js
`Hello, World!\nHostname: ${os.hostname()}\n`
```
you could also potentially cache the hostname above to avoid the function calls on each request, that is perhaps overkill though